charts/intel-gpu-resource-driver/templates/resource-driver.yaml

```diff
@@ -100,9 +79,10 @@ spec:
       - name: sysfs
         hostPath:
           path: /sys
-      - name: defaults
-        configMap:
-          name: defaults
+      # Only use devdri when using fake devfs with device-faker
+      #- name: devdri
+      #  hostPath:
+      #    path: /tmp/test-0123456789/dev/dri
```

Comment:
This does not belong in Helm chart templates, it is only used when testing / deploying DRA driver on nodes without hardware.